src/bitwise/core.cljs

```diff
@@ -161,7 +133,7 @@
        [:div
         [:div
          (str "data: " (util/display-as-binary (get-in @game-state [:resources :data :current])))]
-        [process-list architecture processes]
+        [process-list processes]
```

Comment:
I'd like to figure out a better way to handle this than by passing the architecture all the way down through the application. Maybe the reducer can calculate it and cache it on the process somehow?